extensions/ui-extension/src/hooks/useUpdateAttribute.tsx

```diff
@@ -0,0 +1,37 @@
+import * as Sentry from '@sentry/browser';
+import { useApplyAttributeChange } from '@shopify/ui-extensions-react/checkout';
+import { useEffect } from 'react';
+
+export function useUpdateAttribute(
+   key: string,
+   value: string,
+   skipEmptyValue = false
+): void {
```

Comment:
Minor: it would be nice to accept an object so that the caller has to name the inputs

```suggestion
type UseUpdateAttributeProps = {
   key: string,
   value: string,
   skipEmptyValue?: boolean,
};
export function useUpdateAttribute({
   key,
   value,
   skipEmptyValue = false,
}: UseUpdateAttributeProps): void {
```